src/point.ts

```diff
@@ -17,4 +19,11 @@ export class Point {
 
     get y(): number { return this._y; }
     set y(value: number) { this._y = value; }
+
+    /** @internal */
+    static _create(instance: number): Point {
+        if (instance === 0) return new Point(0, 0);
```

Comment:
I always put the `return` on the next line but I also would not mind doing the style cleanup myself.

Reply:
```javascript
return instance === 0
 ? new Point(0, 0)
 : new Point(
    ImageMagick._api._PointInfo_X_Get(instance),
    ImageMagick._api._PointInfo_Y_Get(instance));
```